scenarios/kubernetes_e2e.py

```diff
@@ -407,6 +407,10 @@ def set_up_aws(workspace, args, mode, cluster, runner_args):
     # TODO(krzyzacy):Remove after retire kops-e2e-runner.sh
     mode.add_aws_runner()
 
+def set_up_acsengine(mode):
+    kubectl_path = check_output('which','kubectl')
```

Comment:
why it uses pre-installed kubectl, rather than kubectl from k8s tarball?

Reply:
It isn't needed anymore. I used it in my test configuration on my prow for speeding up tests. Removed.